tests/workflow/workflow_with_models.py

```diff
@@ -76,3 +78,41 @@
 dict_from_json = json.loads(demo_workflow_json)
 deserialized_demo_workflow = wf.Workflow.dict_to_object(dict_from_json)
 assert demo_workflow == deserialized_demo_workflow
+
+
+# Testing to_script
+demo_workflow.to_script()
```

Comment:
What would be really nice is to test the expected value with the `assert` keyword to check if we get what we expect, instead of only testing if the function is truthy